### DISCUSSION TITLE
Documentation offers to use internal API

### DISCUSSION THREAD

**Discussion by scapital** [cycle/orm#362] [Q&A] [answered]
As I see, [documentation](https://cycle-orm.dev/docs/advanced-typecasting/2.x/en#annotation-typecast-handlers)
offers to use **\Cycle\ORM\Parser\Typecast::class** while configure custom typecast, but [the class](https://github.com/cycle/orm/blob/b69635a37a7eefcf4eb3801013fd62d1e1fcc728/src/Parser/Typecast.php#L15) is marked as internal.

Should I use something else instead of the class or @internal [was added](https://github.com/cycle/orm/pull/260/files#diff-fc3847a1766595eb9034d14ff5cc5d7fdda19fcc0c56d69ad9a02c72395a97ceR13) by mistake?

**Comment by roxblnfk** [comment, marked as answer]
I agree with you. That class should be non-internal. I'll fix it later.
